docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/wysiwyg/common/cgdp-embed-summary-box/index.scss

```diff
@@ -0,0 +1,43 @@
+@use 'uswds-core' as *;
+
+@forward '../../../cgdp-summary-box';
```

Comment:
This should actually import usa-summary-box... cgdp-summary-box, IIRC, is the paragraph type you use on mini-landing pages.